_posts/de/newsletters/2025-06-06-newsletter.md

```diff
@@ -171,4 +171,4 @@ für weitere Informationen zu Payjoin v2.
 [multiprocess project]: https://github.com/ryanofsky/bitcoin/blob/pr/ipc/doc/design/multiprocess.md
 [news320 ipc]: /en/newsletters/2024/09/13/#bitcoin-core-30509
 [news264 payjoin]: /en/newsletters/2023/08/16/#serverless-payjoin
-[news353 pr review]: /de/newsletters/2025/05/09/#bitcoin-core-pr-review-club
```

Comment:
This should have worked, what problem did you see?

Reply:
As discussed out of band, the source of the issue was identified as this feature branch having branched off from master before the German newsletter 353 had been merged which is why the link 404'd in the preview.